filecoin-proofs/tests/api.rs

```diff
@@ -81,25 +81,29 @@ fn seal_lifecycle<Tree: 'static + MerkleTreeTrait>(sector_size: u64) -> Result<(
 #[test]
 #[ignore]
 fn test_winning_post_2kib_base_8() -> Result<()> {
-    winning_post::<SectorShape2KiB>(SECTOR_SIZE_2_KIB)
+    winning_post::<SectorShape2KiB>(SECTOR_SIZE_2_KIB, false);
+    winning_post::<SectorShape2KiB>(SECTOR_SIZE_2_KIB, true)
```

Comment:
Seeing warnings about the `Result` here not being consumed (from each of the first calls)